cli/src/commands.rs

```diff
@@ -0,0 +1,377 @@
+		Commands::ListWorkers => list_workers(cli),
+		Commands::Listen { events, blocks } => listen(cli, events, blocks),
+		Commands::ShieldFunds { from, to, amount, shard } =>
+			shield_funds(cli, from, to, amount, shard),
+		Commands::Trusted(trusted) => trusted_commands::match_trusted_commands(cli, trusted),
+	};
+}
+
+fn balance(cli: &Cli, account: &str) {
+	let api = get_chain_api(cli);
+	let accountid = get_accountid_from_str(account);
+	let balance =
+		if let Some(data) = api.get_account_data(&accountid).unwrap() { data.free } else { 0 };
+	println!("{}", balance);
+}
+
+fn new_account() {
+	let store = LocalKeystore::open(PathBuf::from(&KEYSTORE_PATH), None).unwrap();
+	let key: sr25519::AppPair = store.generate().unwrap();
+	drop(store);
+	println!("{}", key.public().to_ss58check());
+}
+
+fn list_accounts() {
+	let store = LocalKeystore::open(PathBuf::from(&KEYSTORE_PATH), None).unwrap();
+	println!("sr25519 keys:");
+	for pubkey in store.public_keys::<sr25519::AppPublic>().unwrap().into_iter() {
+		println!("{}", pubkey.to_ss58check());
+	}
+	println!("ed25519 keys:");
+	for pubkey in store.public_keys::<ed25519::AppPublic>().unwrap().into_iter() {
+		println!("{}", pubkey.to_ss58check());
+	}
+	drop(store);
+}
+
+fn print_metadata(cli: &Cli) {
+	let meta = get_chain_api(cli).get_metadata().unwrap();
+	println!("Metadata:\n {}", Metadata::pretty_format(&meta).unwrap());
+}
+
+fn print_sgx_metadata(cli: &Cli) {
+	let worker_api_direct = get_worker_api_direct(cli);
+	let metadata = worker_api_direct.get_state_metadata().unwrap();
+	println!("Metadata:\n {}", Metadata::pretty_format(&metadata).unwrap());
+}
+
+fn faucet(cli: &Cli, accounts: &[String]) {
+	let api = get_chain_api(cli);
+	let _api = api.set_signer(AccountKeyring::Alice.pair());
+
+	let mut nonce = _api.get_nonce().unwrap();
+	for account in accounts {
+		let to = get_accountid_from_str(account);
+		#[allow(clippy::redundant_clone)]
+		let xt: UncheckedExtrinsicV4<_> = compose_extrinsic_offline!(
+			_api.clone().signer.unwrap(),
+			Call::Balances(BalancesCall::transfer {
+				dest: GenericAddress::Id(to.clone()),
+				value: PREFUNDING_AMOUNT
+			}),
+			nonce,
+			Era::Immortal,
+			_api.genesis_hash,
+			_api.genesis_hash,
+			_api.runtime_version.spec_version,
+			_api.runtime_version.transaction_version
+		);
+		// send and watch extrinsic until finalized
+		println!("Faucet drips to {} (Alice's nonce={})", to, nonce);
+		let _blockh = _api.send_extrinsic(xt.hex_encode(), XtStatus::Ready).unwrap();
+		nonce += 1;
+	}
+}
+
+fn transfer(cli: &Cli, from: &str, to: &str, amount: &Balance) {
+	let api = get_chain_api(cli);
+	let from_account = get_pair_from_str(from);
+	let to_account = get_accountid_from_str(to);
+	info!("from ss58 is {}", from_account.public().to_ss58check());
+	info!("to ss58 is {}", to_account.to_ss58check());
+	let _api = api.set_signer(sr25519_core::Pair::from(from_account));
```

Comment:
Same thing here.. this is probably just moved code, not written by you? I think it's a small fix though and improves readability.